### PR TITLE
12046 help link on ppr

### PR DESCRIPTION
**Pull request by garyfan1**
*Issue #:* /bcgov/entity#12046

*Description of changes:*
set transition to none, fix some code style

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

**Review by dimak1 — APPROVED** [review]

File: ppr-ui/src/components/tombstone/TombstoneDefault.vue [hunk 1]
```diff
@@ -7,11 +7,14 @@
           <span class="tombstone-header">
             <b>{{ header }}</b>
           </span>
-          <v-tooltip top content-class="top-tooltip pa-5" nudge-left="30">
+          <v-tooltip top content-class="top-tooltip pa-5" nudge-left="30" transition="none">
```
Comment (cameron-eyds) [inline ppr-ui/src/components/tombstone/TombstoneDefault.vue:10]: Does this prop just disable the default transition styling?

**Review by cameron-eyds — APPROVED** [review]
looks good 👍

File: ppr-ui/src/components/tombstone/TombstoneDefault.vue (hunk 1 above)
Comment (garyfan1) [reply ppr-ui/src/components/tombstone/TombstoneDefault.vue:10]: yes, so the tool-tip doesn't fade in or out

Comment (cameron-eyds) [reply ppr-ui/src/components/tombstone/TombstoneDefault.vue:10]: cool, thanks!

**Pull request merged**